src/esp/metadata/attributes/SceneAttributes.cpp

```diff
@@ -18,10 +18,28 @@ const std::map<std::string, esp::physics::MotionType>
 SceneObjectInstanceAttributes::SceneObjectInstanceAttributes(
     const std::string& handle)
     : AbstractAttributes("SceneObjectInstanceAttributes", handle) {
-  setMotionType(-1);  // defaults to unknown
+  // defaults to unknown/undefined
+  setMotionType(static_cast<int>(esp::physics::MotionType::UNDEFINED));
```

Comment:
This is only used to override the default setting already in place.  So this comes into play in the following code from Simulator.cpp (466) :

```
    const physics::MotionType attrObjMotionType =
        static_cast<physics::MotionType>(objInst->getMotionType());
    const physics::MotionType objMotionType =
        physicsManager_->getObjectMotionType(objID);

    if ((attrObjMotionType != objMotionType) &&
        (attrObjMotionType != physics::MotionType::UNDEFINED)) {
      physicsManager_->setObjectMotionType(objID, attrObjMotionType);
    }
```

Reply:
..which has been changed heh.  Now it will only attempt to change the value if the object instance motion type value is not undefined.
```
    // set object's motion type if different than set value
    const physics::MotionType attrObjMotionType =
        static_cast<physics::MotionType>(objInst->getMotionType());
    if (attrObjMotionType != physics::MotionType::UNDEFINED) {
      physicsManager_->setObjectMotionType(objID, attrObjMotionType);
    }
```